pkg/api/mlflow/service/run/service.go

```diff
@@ -156,6 +157,7 @@ func (s Service) GetRun(
 func (s Service) SearchRuns(
 	ctx context.Context, namespace *models.Namespace, req *request.SearchRunsRequest,
 ) ([]models.Run, int, int, error) {
+	adjustSearchRunsRequestForNamespace(namespace, req)
```

Comment:
since we have normal `builder` functions which will convert `req` into `db` query, here it looks temporary reasonable to have it.

Reply:
maybe it is reasonable to firstly validate request and then do all the adjustments?

Reply:
okay, moved below the validation step

Reply:
done